src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  min-width: 1024px;
+  max-width: 1200px;
```

Comment:
it's not a good practice to use such rules at html document. You can work with header sizes
```suggestion
```